spec/shippo/api/model/batch_spec.rb

```diff
@@ -0,0 +1,143 @@
+require 'dummy_helper'
+require 'spec_helper'
+
+DEFAULT_CARRIER_ACCOUNT = 'bb980a0dcd2047328024bc3dcad56682'
+DEFAULT_SERVICELEVEL_TOKEN = 'usps_priority'
+
+# Retries up to 10 times to retrieve a valid
+# Batch (which takes time to become 'VALID' from
+# 'VALIDATING' after initially being created).
+def retrieve_valid_batch(id)
+  retries = 10
+  retrieve = nil
+  until retries == 0 do
+    sleep 1
+    retrieve = Shippo::Batch::get(id)
+    break if retrieve[:object_status] == 'VALID'
+    retries -= 1
+  end
+  STDERR.puts 'Unable to retrieve VALID Batch object' unless retrieve
+  retrieve
+end
+
+RSpec.describe 'Shippo::API::Batch' do
+  let(:dummy_batch) { DUMMY_BATCH }
+  let(:dummy_shipment) { DUMMY_SHIPMENT }
+
+  describe '#create' do
+    it 'should properly create and return a Batch object' do
```

Comment:
Added new invalid create test for Batch. There's no create API for Track currently, just retrieval and registering a webhook: https://goshippo.com/docs/reference/#tracks